app/web_hub.go

```diff
@@ -178,7 +178,8 @@ func (s *Server) Publish(message *model.WebSocketEvent) {
 			message.EventType() == model.WebsocketEventPostEdited ||
 			message.EventType() == model.WebsocketEventDirectAdded ||
 			message.EventType() == model.WebsocketEventGroupAdded ||
```

Comment:
Maybe we can add the broadcast type as reliable for all of these messages and then remove these from the if condition altogether? In a separate HW ticket of course.

Reply:
That's a nice idea and it probably makes sense to let the higher layer decide how they should get sent.